### ISSUE TITLE
confusion about the paper's formulas 

### ISSUE THREAD

**Issue by weiqingcai** [takashiishida/pconf#3]
Hi @takashiishida Thank you very much for your reply. If I didn't understand it wrong, you mean 
![formula2](https://user-images.githubusercontent.com/10105321/70600845-b8557080-1c2b-11ea-9dee-c460858fd87e.png).
But i can only get ![formula3](https://user-images.githubusercontent.com/10105321/70601658-88a76800-1c2d-11ea-8902-50c3af7a7c4f.png)

Another question is, when I replace the formula in the code`logistic = -1. * negative_logistic(-1. * model(inputs)) loss = torch.sum(-model(inputs)+ logistic * 1. / confidence)` with the formula in the paper`loss = torch.sum(negative_logistic(model(inputs)) + negative_logistic(-model(inputs))*(1.-confidence)/confidence)`, the results are different. What is the reason?Why the results obtained by directly optimizing the formulas in the paper are poor？

**Comment by takashiishida** [comment]
I'm sorry but there was a typo in my post in Issue #2 so the correct proof would be:
\ell_L(g) - \ell_L(-g)
= \log \frac{1+exp(-g)}{1+exp(g)}
= \log \frac{1+e^{-g}}{1+e^{g}}
= \log e^{-g}
= -g

Mathematically the two equations are equivalent and should give similar experimental results.  I haven't tried the experiments with the latter implementation, so I will need to investigate to see if there are some numerical issues.

**Comment by weiqingcai** [comment]
Thank you very much for your patience👍
